apps/prairielearn/src/lib/github.js

```diff
@@ -289,15 +289,15 @@ export async function createCourseRepoJob(options, authn_user) {
     });
 
     job.info('Sync git repository to database');
-    const sync_result = await syncDiskToSql(inserted_course.id, inserted_course.path, job);
+    const syncResult = await syncDiskToSql(inserted_course.id, inserted_course.path, job);
```

Comment:
I was trying to find all the places `syncDiskToSql` was called and it was rather annoying that the return value was named many different things in many different places, so I tried editing for consistency.

I didn't add any new checks here, since it is always assumed that this particular sync will succeed (the code also doesn't have any checks for `hadJsonErrors` at this point)

Reply:
I think it's reasonable to not add any new checks here. At the same time, if you move the check into `syncDiskToSql` or whatever as discussed in my other comment, it doesn't hurt to add it.